mesonbuild/backend/ninjabackend.py

```diff
@@ -1882,6 +1884,13 @@ def generate_single_compile(self, target, outfile, src, is_generated=False, head
             raise AssertionError('BUG: sources should not contain headers {!r}'.format(src))
         if isinstance(src, RawFilename) and src.fname.endswith('.h'):
             raise AssertionError('BUG: sources should not contain headers {!r}'.format(src.fname))
+        extra_orderdeps = []
```

Comment:
__[flake8]__
* [F841] local variable 'extra_orderdeps' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->